CHANGES/1812.feature

```diff
@@ -0,0 +1 @@
+Add limited v1 support for legacy roles for hosted galaxy instance (tech preview)
```

Comment:
I don't want customers to see this and think "great, we're getting role support for on prem". I would maybe change this to misc so it doesn't show up in the changelog unless you click on it, or maybe specifically say beta.galaxy.ansible.com in the notes (hosted galaxy instance may not be clear to people who don't know our project well)

Reply:
Agreed, I want this to be clear this is for community galaxy. I updated the text.